client/landing/gutenboarding/onboarding-block/signup-form/index.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+// /**
+//  * Internal dependencies
+//  */
+import {
+	usePasswordlessSignUp,
+	UsePasswordlessSignUpStatus,
+	Provider,
+} from '@automattic/authentication';
+import { Button, TextControl, Modal } from '@wordpress/components';
+
+import './style.scss';
+
+const SignupForm = () => {
+	const [ emailVal, setEmailVal ] = useState( '' );
+	const { signUp, status, error } = usePasswordlessSignUp();
+
+	const handleSignUp = ( event: React.FormEvent< HTMLFormElement > ) => {
+		event.preventDefault();
+		signUp( emailVal );
+	};
+
+	const renderTosLink = () => {
+		return (
+			<a href="https://wordpress.com/tos/" target="_blank" rel="noopener noreferrer">
```

Comment:
[`ExternalLink`](https://github.com/WordPress/gutenberg/tree/8e7e67bae2d712120137ca86977c10994e8c637a/packages/components/src/external-link#readme) component might be useful here.